### PR TITLE
Separate methods for formatted and unformatted log

### PR DESCRIPTION
**Pull request by arknable**

### CODE REVIEW

File: aws/aws_instance.go [hunk 1]
```diff
@@ -218,20 +218,20 @@ func (p *AWS) CreateInstance(ctx *lepton.Context) error {
...
-		ctx.Logger().Debug("creating new security group in vpc %s", *vpc.VpcId)
+		ctx.Logger().Debugf("creating new security group in vpc %s", *vpc.VpcId)
 		sg, err = p.CreateSG(ctx, svc, imgName, *vpc.VpcId)
 		if err != nil {
 			return err
 		}
 	}
 
-	ctx.Logger().Debug("getting subnet")
+	ctx.Logger().Debugf("getting subnet")
```
Comment (eyberg) [inline aws/aws_instance.go:234]: not formatted?

File: aws/aws_instance.go [hunk 2]
```diff
@@ -273,18 +273,18 @@ func (p *AWS) CreateInstance(ctx *lepton.Context) error {
...
+		log.Errorf("Could not create instance %v", err)
 		return err
 	}
 
 	log.Info("Created instance", tagInstanceName)
 
 	// create dns zones/records to associate DNS record to instance IP
 	if cloudConfig.DomainName != "" {
-		ctx.Logger().Debug("associating domain to the created instance")
+		ctx.Logger().Debugf("associating domain to the created instance")
```
Comment (eyberg) [inline aws/aws_instance.go:287]: not formatted?

File: aws/aws_instance.go (hunk 1 above)
Comment (eyberg) [inline aws/aws_instance.go:234]: not formatted?

File: aws/aws_instance.go (hunk 2 above)
Comment (eyberg) [inline aws/aws_instance.go:287]: not formatted?

**Review by eyberg — CHANGES_REQUESTED** [review]
just a few spots with non-formatted logs

File: aws/aws_instance.go (hunk 1 above)
Comment (arknable) [reply aws/aws_instance.go]: Fixed by https://github.com/nanovms/ops/pull/1102/commits/2920037fc326cf982510341c6c127f312569fa73

File: aws/aws_instance.go (hunk 2 above)
Comment (arknable) [reply aws/aws_instance.go]: Fixed by https://github.com/nanovms/ops/pull/1102/commits/2920037fc326cf982510341c6c127f312569fa73

File: aws/aws_instance.go (hunk 1 above)
Comment (arknable) [reply aws/aws_instance.go]: Fixed by https://github.com/nanovms/ops/pull/1102/commits/2920037fc326cf982510341c6c127f312569fa73

File: aws/aws_instance.go (hunk 2 above)
Comment (arknable) [reply aws/aws_instance.go]: Fixed by https://github.com/nanovms/ops/pull/1102/commits/2920037fc326cf982510341c6c127f312569fa73

**Review by eyberg — APPROVED** [review]

**Comment by arknable** [comment]
Rebased and squashed

**Pull request merged**